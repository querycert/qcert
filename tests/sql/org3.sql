select name from (select * from employees union select * from students) as persons where age = 32;
