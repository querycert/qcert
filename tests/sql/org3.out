[{"name": "John Doe"}, {"name": "Jane Doe"}, {"name": "Jill Does"}, {"name": "Tim Coe"}]